cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -25,6 +30,71 @@
 log = logging.getLogger('edx.modulestore')
 
 
+class AboutInfo(object):
+    """ About info structure to contain
+       1) Property name to use
+       2) Where to add in the index (using flags above)
+       3) Where to source the properties value
+    """
+    # Bitwise Flags for where to index the information
+    #
+    # ANALYSE - states that the property text contains content that we wish to be able to find matched within
+    #   e.g. "joe" should yield a result for "I'd like to drink a cup of joe"
+    #
+    # PROPERTY - states that the property text should be a property of the indexed document, to be returned with the
+    # results: search matches will only be made on exact string matches
+    #   e.g. "joe" will only match on "joe"
+    #
+    # We are using bitwise flags because one may want to add the property to EITHER or BOTH parts of the index
+    #   e.g. university name is desired to be analysed, so that a search on "Oxford" will match
+    #   property values "University of Oxford" and "Oxford Brookes University",
+    #   but it is also a useful property, because within a (future) filtered search a user
+    #   may have chosen to filter courses from "University of Oxford"
+    #
+    # see https://wiki.python.org/moin/BitwiseOperators for information about bitwise shift operator used below
+    #
+    ANALYSE = 1 << 0  # Add the information to the analysed content of the index
+    PROPERTY = 1 << 1  # Add the information as a property of the object being indexed (not analysed)
+
+    # Source location options - this is the function that returns the value of the property, signature should be
+    #   course, attribute_name, modulestore
+    @staticmethod
+    def fetch_from_about(course, attribute_name, modulestore):
+        """ Get about attribute from modulestore """
+        usage_key = course.id.make_usage_key('about', attribute_name)
+        try:
+            value = modulestore.get_item(usage_key).data
```

Comment:
Thanks @jimabramson:
* Unfortunately, about data does not appear to be present in course loaded with depth=None because they are not children of the object
* Yes, thanks - `get_items` seems to have done the trick!
* I shall wrap it up!